usaspending_api/common/serializers.py

```diff
@@ -12,6 +12,20 @@ def __init__(self, *args, **kwargs):
         exclude_fields = kwargs.pop('exclude', None)
         super(LimitableSerializer, self).__init__(*args, **kwargs)
 
+        # Create and initialize the child serializers
+        try:
+            # Initialize the child serializers
+            children = self.Meta.nested_serializers
+            for field in children.keys():
+                child_args = {
+                    **children[field].get("kwargs", {}),
+                    "context": self.context
+                }
+                self.fields[field] = children[field]["class"](**child_args)
+        except:
```

Comment:
I think it should be fine as the list serializer will have a list of LimitableSerializers, we'll just need to override many_init to add a line to pass the context to the children.

Yeah, we should catch specifically AttributeNotFound exceptions here. I will update that.